prow/cmd/deck/static/script.js

```diff
@@ -138,6 +138,8 @@ function redraw() {
         r.appendChild(stateCell(build.state));
         if (build.pod_name) {
             r.appendChild(createLinkCell("\u2261", "log?pod=" + build.pod_name));
+        } else if (build.url !== "") {
+            r.appendChild(createLinkCell("\u2261", build.url));
```

Comment:
emmm, the goal is to have some log place for finished pods. Maybe the below line is enough, or I can use a different icon here.